fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/communication/CellModule.java

```diff
@@ -152,6 +153,15 @@ public void sendV2xMessage(V2xMessage msg) {
      * @return the created builder for further configuration
      */
     public CellMessageRoutingBuilder createMessageRouting() {
-        return new CellMessageRoutingBuilder(getOwner().getId(), getOwner().getPosition());
+        GeoPoint position;
+        try {
+            position = getOwner().getPosition();
+        } catch (UnsupportedOperationException e) {
+            if (log.isDebugEnabled()) {
+                log.debug("Cell Message built without source position: {}", e.getMessage());
+            }
```

Comment:
log.error("Cell Message built without source position", e);

Reply:
I don't think this should be an error, since servers and tmc's built all their cell messages without a position.

Reply:
The it would be better to have 
`position = getOwner() instanceof ServerOperatingSystem ? null : getOwner().getPosition()` and do not catch any error

Reply:
Fixed.